src/sentry/incidents/models.py

```diff
@@ -203,7 +203,10 @@ class Meta:
         app_label = "sentry"
         db_table = "sentry_incident"
         unique_together = (("organization", "identifier"),)
-        index_together = (("alert_rule", "type", "status"),)
+        index_together = (
+            ("alert_rule", "type", "status"),
+            ("id", "date_added"),
```

Comment:
@isabellaenriquez wondering why we'd make this index, which query does this refer to?

Reply:
`trigger.triggered_incidents.order_by("-date_added").first()`

Which produces
```
SELECT *
FROM "sentry_incident" 
INNER JOIN "sentry_incidenttrigger" ON ("sentry_incident"."id" = "sentry_incidenttrigger"."incident_id") 
WHERE "sentry_incidenttrigger"."alert_rule_trigger_id" = 1 
ORDER BY "sentry_incident"."date_added" DESC
```

Reply:
Although I'm not totally sure how useful it will be now that I'm thinking it through more, since there will be only one date_added per incident id...

Reply:
hm, so are the indexes still worth adding?

Reply:
Yeah, let's give it a try, it at least means we can scan the index rather than the table to sort. We can see how effective it is in redash https://redash.getsentry.net/queries/5173/source (this trigger has 130k+ plus incidents and so is a good test case)